modules/ROOT/pages/managing-files.adoc

```diff
@@ -50,3 +50,33 @@ storage:
       target: /opt/tools/transmogrifier
       hard: false
 ----
+
+.Example for how to set permissions and ownership for a file and its parent directories
+[source,yaml]
+----
+variant: fcos
+version: 1.4.0
+storage:
+  files:
+      # This creates a file, and sets the ownership
+    - path: /home/directory/you/want.txt
```

Comment:
Let's use a path with a name that matches the user and use a real configuration example like https://discussion.fedoraproject.org/t/ignition-when-a-file-is-created-parent-dir-is-still-owned-by-root/41222